pkg/distributor/distributor_test.go

```diff
@@ -1710,9 +1710,43 @@ func TestDistributor_ExemplarValidation(t *testing.T) {
 				},
 			},
 			expectedMetrics: `
+		        # HELP cortex_discarded_exemplars_total The total number of exemplars that were discarded.
+		        # TYPE cortex_discarded_exemplars_total counter
+		        cortex_discarded_exemplars_total{reason="exemplar_too_far_in_future",user="user"} 1
+		    `,
+		},
+		"should drop exemplars more than the allowed exemplar per serie limit, when multiple exemplars are specified in the same series": {
```

Comment:
```suggestion
		"should drop exemplars more than the allowed exemplar per series limit, when multiple exemplars are specified in the same series": {
```

Reply:
I think "should drop exemplars above" is more descriptive than "should drop exemplars more".